arctic/_util.py

```diff
@@ -23,7 +23,7 @@ def are_equals(o1, o2, **kwargs):
         return False
 
 
-def enable_sharding(arctic, library_name, hashed=True):
+def enable_sharding(arctic, library_name, hashed=True, key='symbol'):
```

Comment:
are other keys possible? are they all user defined? are they all internally defined? I can't tell. Might be nice to add a docstring to this